### PR TITLE
Remove incorrect formatting, unused conditions

### PR DESCRIPTION
**Pull request by yah01**
Signed-off-by: yah01 <yang.cen@zilliz.com>
/kind improvement

### CODE REVIEW

**Comment by yah01** [comment]
rerun ut

File: tests/python_client/testcases/test_delete.py [hunk 1]
```diff
@@ -1781,8 +1692,7 @@ def test_delete_invalid_expr(self):
         expected: Raise exception
         """
         collection_w = \
-            self.init_collection_general(
-                prefix, nb=tmp_nb, insert_data=True, primary_field=ct.default_string_field_name)[0]
+            self.init_collection_general(    prefix, nb=tmp_nb, insert_data=True, primary_field=ct.default_string_field_name)[0]
```
Comment (binbinlv) [inline tests/python_client/testcases/test_delete.py:1695]: Could you please delete the blanks before "prefix"? Thanks.

File: tests/python_client/testcases/test_delete.py [hunk 2]
```diff
@@ -1132,15 +1070,13 @@ def test_delete_sealed_only(self):
...
-            name=cf.gen_unique_str(prefix), shards_num=2)
+        collection_w = self.init_collection_wrap(name=cf.gen_unique_str(prefix), shards_num=2)
         # insert 3000 entities into 3 segments
         segment_num = 3
         segment_per_count = 2000
         ids = []
         for i in range(segment_num):
-            df = cf.gen_default_dataframe_data(
-                nb=segment_per_count, start=(i * segment_per_count))
+            df = cf.gen_default_dataframe_data(    nb=segment_per_count, start=(i * segment_per_count))
```
Comment (binbinlv) [inline tests/python_client/testcases/test_delete.py:1079]: Could you please delete the blanks before "nb"? Thanks.

File: tests/python_client/testcases/test_delete.py [hunk 3]
```diff
@@ -870,14 +819,12 @@ def test_delete_insert_same_id_sealed(self, to_query):
...
-        collection_w.query(
-            tmp_expr, check_task=CheckTasks.check_query_results, check_items={'exp_res': res})
+        collection_w.query(tmp_expr, check_task=CheckTasks.check_query_results, check_items={'exp_res': res})
 
         # delete
         collection_w.delete(tmp_expr)
         if to_query:
-            collection_w.query(
-                tmp_expr, check_task=CheckTasks.check_query_empty)
+            collection_w.query(    tmp_expr, check_task=CheckTasks.check_query_empty)
```
Comment (binbinlv) [inline tests/python_client/testcases/test_delete.py:827]: Could you please delete the blanks before "tmp_expr"? Thanks.

File: tests/python_client/testcases/test_delete.py [hunk 4]
```diff
@@ -825,8 +776,7 @@ def test_delete_insert_same_id_growing(self, to_query, to_flush):
         del_res, _ = collection_w.delete(tmp_expr)
         log.debug(f'to_query:{to_query}')
         if to_query:
-            collection_w.query(
-                tmp_expr, check_task=CheckTasks.check_query_empty)
+            collection_w.query(    tmp_expr, check_task=CheckTasks.check_query_empty)
```
Comment (binbinlv) [inline tests/python_client/testcases/test_delete.py:779]: Could you please delete the blanks before "tmp_expr"? Thanks.

File: tests/python_client/testcases/test_delete.py [hunk 5]
```diff
@@ -1335,8 +1262,7 @@ def test_delete_expr_repeated_values_with_string(self):
         """
         # init collection with nb default data
         collection_w = \
-            self.init_collection_general(
-                prefix, nb=tmp_nb, insert_data=True, primary_field=ct.default_string_field_name)[0]
+            self.init_collection_general(    prefix, nb=tmp_nb, insert_data=True, primary_field=ct.default_string_field_name)[0]
```
Comment (binbinlv) [inline tests/python_client/testcases/test_delete.py:1265]: Could you please delete the blanks before "prefix"? Thanks.

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**